README.md

```diff
@@ -175,13 +178,16 @@ Ascii Graph, like coloring in terminal, so please use `json`  or `yaml` instead.
 
 <details>
 
-<summary>metrics multiple-linear [--start=start-time] [--end=end-time] --name=metrics-name [--service=service-name] [--num=number-of-linear-metrics]</summary>
+<summary>metrics multiple-linear [--start=start-time] [--end=end-time] --name=metrics-name [--service=service-name] [--num=number-of-linear-metrics] [--instance=instance-name] [--endpoint=endpoint-name] [--isNormal=true/false]</summary>
```

Comment:
The label of mutiple-linear supports more scenarios more than the percentile. I suggest changing `num` to `label`.

Reply:
Agree.

Reply:
> The label of mutiple-linear supports more scenarios more than the percentile. I suggest changing `num` to `label`.

Then it will like `--label="0,1,2,3,4"`, right ? Otherwise we need define the format of `label`.

Reply:
`--label="0,1,2,3,4"` looks good.

Reply:
Done